k3d.yaml

```diff
@@ -59,12 +59,16 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/etc/confd
           cp -ar templates "${{targets.subpkgdir}}"/etc/confd/
           cp -ar conf.d "${{targets.subpkgdir}}"/etc/confd/
-      - uses: git-checkout
+      - name: git-chekout-confd # There is not a new release in last 6 years
+        runs: |
+          git clone https://github.com/kelseyhightower/confd.git /home/confd
```

Comment:
We are changing this but the source repository doesn't use this code. I am sure we want to replace their confd with a different source code.

Reply:
The source repo is using a fork of this with basically no change. I have checked They have just added go module reference [commit](https://github.com/kelseyhightower/confd/commit/b974b1eef60fa7c6fb5b0d2842d669d9eed96375#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6) which was not there in the original on that time.

I think it is okay to change.

Reply:
It is good that you checked the difference between both. But my main concern comes from using something that isn't used in the source repository. How can we track any new rollouts or changes on `github.com/kelseyhightower/confd` and keep compatible to changes on k3d here ?

Reply:
Yes and I have the same concern, we will not get known if there is any changes. And I am not sure what should we do here. Let me see if I can do go-bump to all the dependency.

Reply:
Were you able to use go-bump ?

Reply:
I have tried but still could not get it to work. Several modules version mismatchd.

Reply:
I'm not sure how `go-bump` would work here.

Another option would be to create a new package for `confd`, and have a runtime dep on it. It's kinda weird how we package confd directly inside k3d

Reply:
I agree with @pdeslaur 100% here. Let's avoid packing confd here as a subpackage.

Reply:
@hectorj2f  - how about we merge this CVE fix, and change the packaging as a 2 step process?

Reply:
lgtm